bukkit/build.gradle

```diff
@@ -0,0 +1,75 @@
+plugins {
+    id 'com.github.johnrengelman.shadow' version '2.0.4'
+}
+
+repositories {
+    maven { url 'https://hub.spigotmc.org/nexus/content/repositories/snapshots/' }
+    maven { url 'https://libraries.minecraft.net/' }
+}
+
+dependencies {
+    compile project(':common')
+
+    compileOnly 'org.spigotmc:spigot-api:1.14-R0.1-SNAPSHOT'
+    compileOnly 'net.kyori:text-adapter-bukkit:3.0.2'
+    compileOnly 'me.lucko:commodore:1.3'
+    compileOnly('net.milkbowl.vault:VaultAPI:1.6') {
+        exclude(module: 'bukkit')
+    }
+    compileOnly 'lilypad.client.connect:api:0.0.1-SNAPSHOT'
+
+    // migration plugins
+    compileOnly 'org.tyrannyofheaven.bukkit:zPermissions:1.3'
+    compileOnly('ru.tehkode:PermissionsEx:1.23.5') {
+        exclude(module: 'bukkit')
+        exclude(module: 'updater')
+        exclude(module: 'commons-dbcp')
+        exclude(module: 'AccountsClient')
+    }
+    compileOnly 'com.github.gustav9797:PowerfulPermsAPI:4.5.2'
+    compileOnly 'org.anjocaido:GroupManager:1.4'
+    compileOnly 'de.bananaco:bpermissions-api:2.12'
+    compileOnly('com.platymuus:bukkit-permissions:2.5') {
+        exclude(module: 'bukkit')
+        exclude(module: 'metrics')
+    }
+
```

Comment:
None of the permission dependencies needed

Reply:
Yeah a lot of the build gradle parts can be cleaned up, i basically just copied luckperms for the time being